docs/tutorial/working-with-instancegroups.md

```diff
@@ -234,6 +234,29 @@ spec:
   rootVolumeIops: 200
 ```
 
+## Encrypting the root volume
+
+As of kOps 1.19.0 you can encrypt the root volume  _(note, presently confined to AWS)_ via the instancegroup specification.
```

Comment:
The kops_feature_table macro could be used here

Reply:
Done. :)